tests/testthat/_snaps/stress_test_arguments.md

```diff
@@ -1,23 +1,13 @@
 # hasn't changed
 
     Code
-      stress_test_arguments
+      str(stress_test_arguments)
     Output
-      # A tibble: 14 x 6
-         name                    type      default       allowed           min   max  
-         <chr>                   <chr>     <chr>         <chr>             <chr> <chr>
-       1 asset_type              character <NA>          equity, bonds, l~ <NA>  <NA> 
-       2 baseline_scenario       character WEO2021_STEPS WEO2021_STEPS, G~ <NA>  <NA> 
-       3 shock_scenario          character WEO2021_SDS   WEO2021_SDS, WEO~ <NA>  <NA> 
-       4 lgd                     double    0.45          <NA>              0.3   0.9  
-       5 risk_free_rate          double    0.02          <NA>              0     0.05 
-       6 discount_rate           double    0.07          <NA>              0.015 0.1  
-       7 growth_rate             double    0.03          <NA>              0.01  0.099
-       8 div_netprofit_prop_coef double    1             <NA>              0.8   1    
-       9 shock_year              double    2030          <NA>              2025  2035 
-      10 scenario_geography      character Global        Africa, AsiaPaci~ <NA>  <NA> 
-      11 start_year              double    2021          <NA>              2021  2021 
-      12 settlement_factor       double    1             <NA>              0     1    
-      13 exp_share_damages_paid  double    0.027         <NA>              0     1    
-      14 scc                     double    40            <NA>              0     10000
+      tibble [14 x 6] (S3: tbl_df/tbl/data.frame)
+       $ name   : chr [1:14] "asset_type" "baseline_scenario" "shock_scenario" "lgd" ...
```

Comment:
unfortunately this sort of made id worse.. you can see from the ... that it is still truncating, this time rows, and I would assume that is evene worse... How about removing this change for now (use git revert if you want to be fancy), merging the PR and looking at the other options that dear @maurolepore  suggested in a follow up?